### PR TITLE
chore: add the ability to handle new metrics event in gateway

### PR DESCRIPTION
**Pull request by ono-max**

### CODE REVIEW

File: pkg/gateway/api/validation.go [hunk 1]
```diff
@@ -127,3 +134,82 @@ func (s *gatewayService) validateMetricsEvent(ctx context.Context, id string) (s
...
+	return "", nil
+}
+
+func (s *gatewayService) validateInternalSdkErrorMetricsEvent(
+	ctx context.Context,
+	apiID eventproto.ApiId,
+) (string, error) {
+	if apiID == eventproto.ApiId_UNKNOWN_API {
+		s.logger.Warn(
+			"Failed to validate latencyMetrics event. Unkonown API id",
```
Comment (masaaania) [inline pkg/gateway/api/validation.go:210]: All warn messages are "Failed to validate `latencyMetrics` event. Unknown API id".

Comment (ono-max) [reply pkg/gateway/api/validation.go:210]: Good catch

**Comment by cre8ivejp** [comment]
Please fix the lint errors tomorrow.

**Review by cre8ivejp — APPROVED** [review]
Nice!
Thanks!

**Pull request merged**